packages/sil-governance/src/routes/create-proposal/index.tsx

```diff
@@ -24,14 +24,20 @@ const CreateProposal = (): JSX.Element => {
   const [electorates, setElectorates] = useState<readonly Electorate[]>([]);
 
   useEffect(() => {
+    let isSubscribed = true;
+
     const updateElectorates = async (): Promise<void> => {
       // in DOM tests, governor is not set
-      if (governor) {
+      if (governor && isSubscribed) {
         const electorates = await governor.getElectorates();
         setElectorates(electorates);
```

Comment:
No, you should wrap `setElectorates(electorates)` inside `isSubscribed` condition, because you have `async` operation inside current block:
```ts
if (governor) {
  const electorates = await governor.getElectorates();
  if (isSubscribed) {
    setElectorates(electorates);
  }
}
```

Reply:
I already had tried this but both approaches give the same warning. I have taken a look at the show-electorate route in order to isolate it, which has very few lines of code and produces the warning and I don't know what may be wrong.